cloud-reactor-api/src/main/java/com/sequenceiq/cloudbreak/cloud/event/instance/StopStartUpscaleStartInstancesRequest.java

```diff
@@ -0,0 +1,66 @@
+package com.sequenceiq.cloudbreak.cloud.event.instance;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+
+import com.sequenceiq.cloudbreak.cloud.context.CloudContext;
+import com.sequenceiq.cloudbreak.cloud.event.resource.CloudStackRequest;
+import com.sequenceiq.cloudbreak.cloud.model.CloudCredential;
+import com.sequenceiq.cloudbreak.cloud.model.CloudInstance;
+import com.sequenceiq.cloudbreak.cloud.model.CloudStack;
+
+public class StopStartUpscaleStartInstancesRequest<T> extends CloudStackRequest<T> {
```

Comment:
StopStartUpscaleStartInstancesResult instead of T?

Reply:
Will change this.

Reply:
Fixed via CB-15827